docs/Setup.rst

```diff
@@ -1,3 +1,11 @@
 *****
 Setup
 *****
+
+The Ignition framework is a .NET MVC based architecture development framework used to create a coded infrastructure to support a Sitecore based website.   Sitecore is a content management website development framework.  Ignition provides a rapid development mechanism for creating components used to render pages using Sitecore as the website framework.
```

Comment:
I don't think this is necessary in the _Setup_ section as it's already in the index of the docs.